examples/bq-log-alerting/main.tf

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  credentials_file_path = var.credentials_path
+}
+
+/*****************************
+  Provider configuration
+ ****************************/
+provider "google" {
+  credentials = file(local.credentials_file_path)
+  version     = "~> 3.30"
+}
+
+module "bq-log-alerting" {
+  source          = "../..//modules/bq-log-alerting"
```

Comment:
typo?
```suggestion
  source          = "../../modules/bq-log-alerting"
```

Reply:
It works with this format and this format of adding `//` before `modules` is used in all the examples:

https://github.com/terraform-google-modules/terraform-google-log-export/blob/e8229b57a41c18b80748b11b8251ddc814ad71ef/examples/bigquery/folder/main.tf#L34

https://github.com/terraform-google-modules/terraform-google-log-export/blob/e8229b57a41c18b80748b11b8251ddc814ad71ef/examples/storage/folder/main.tf#L35

https://github.com/terraform-google-modules/terraform-google-log-export/blob/e8229b57a41c18b80748b11b8251ddc814ad71ef/examples/pubsub/folder/main.tf#L34

Reply:
Oh interesting! TIL 😄